src/cryptocontextimpl.cpp

```diff
@@ -124,6 +124,17 @@ void wrap_CryptoContextImpl(jlcxx::Module& mod) {
                                       lbcrypto::ConstCiphertext<lbcrypto::DCRTPoly>) const>(&WrappedT::EvalSub));
 
         wrapped.method("EvalMultKeyGen", &WrappedT::EvalMultKeyGen);
+        // static_cast is not applicable to static functions, so wrap overloaded static function
+        // using lambda expressions
+        wrapped.method("ClearEvalMultKeys", [](){
+            return WrappedT::ClearEvalMultKeys();
+          });
+        wrapped.method("ClearEvalMultKeys", [](const lbcrypto::CryptoContext<lbcrypto::DCRTPoly> cc){
+            return WrappedT::ClearEvalMultKeys(cc);
+          });
+        wrapped.method("ClearEvalMultKeys", [](const std::string& id){
+            return WrappedT::ClearEvalMultKeys(id);
+          });
```

Comment:
Could something like the following work?
Why doesn't something like
```cpp
wrapped.method("ClearEvalMultKeys", static_cast<void(*)()>(&WrappedT::ClearEvalMultKeys));
wrapped.method("ClearEvalMultKeys", static_cast<void(*)(const lbcrypto::CryptoContext<lbcrypto::DCRTPoly>)>(&WrappedT::ClearEvalMultKeys));
wrapped.method("ClearEvalMultKeys", static_cast<void(*)(const std::string&)>(&WrappedT::ClearEvalMultKeys));
```
work?

Reply:
Good idea!
It doesn't work while wrapping `CryptoContextImpl` class, but works, if these methods are wrapped separately, e.g.
```
typedef lbcrypto::CryptoContextImpl<lbcrypto::DCRTPoly> WrappedT;    
mod.method("ClearEvalMultKeys", static_cast<void (*)()>(&WrappedT::ClearEvalMultKeys));
mod.method("ClearEvalMultKeys",
                      static_cast<void (*)(const lbcrypto::CryptoContext<lbcrypto::DCRTPoly>)>(&WrappedT::ClearEvalMultKeys));
mod.method("ClearEvalMultKeys", static_cast<void (*)(const std::string&)>(&WrappedT::ClearEvalMultKeys));
```